nest/core/pynest_factory.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import FastAPI
+from nest.core.pynest_application import PyNestApp
+from typing import TypeVar, Union
+from nest.core.pynest_container import PyNestContainer
+
+T = TypeVar("T")
+
+
+class PyNestFactory:
+    @staticmethod
+    def _initialize(
+        module: T, container: PyNestContainer,
+    ):
+        container.add_module(module)
+
+    @staticmethod
+    def create(main_module: T, **kwargs) -> PyNestApp:
+        container = PyNestContainer()
+        http_server = PyNestFactory._create_server(**kwargs)
+        PyNestFactory._initialize(main_module, container)
+        app = PyNestApp(container, http_server)
+        return app
```

Comment:
return app.get_server()

Reply:
By doing that, we'll still use the FastAPI approach with `@app.get()`, for example.